frontend/src/common/ReviewWithComment.tsx

```diff
@@ -135,6 +145,7 @@ export default function ReviewWithComment({
               value={reviewRequest}
               getOptionLabel={(option) => getRoleDisplay(option.role, modelRoles)}
               options={reviews}
+              defaultValue={reviewRequest}
```

Comment:
Sorry for not making this clear, but `defaultValue` isn't required at all. We're already controlling the value with the `value` prop